apps/docs/app/features/interactive-code/LiveEditor.tsx

```diff
@@ -9,18 +9,72 @@ type LiveEditorProps = Omit<BoxProps, "onChange"> & {
 };
 export const LiveEditor = ({ onChange, ...props }: LiveEditorProps) => {
   const liveContext = useContext(LiveContext);
+  const containerRef = useRef<HTMLDivElement>(null);
+  const [isDisabled, setDisabled] = useState(true);
 
   if (!liveContext.code) {
     return null;
   }
 
   return (
-    <CodeBlockContainer code={liveContext.code} spellCheck={false} {...props}>
-      <ReactLiveEditor
-        theme={theme}
-        style={{ overflow: "scroll", height: "100%" }}
-        {...(onChange ? { onChange } : {})}
-      />
-    </CodeBlockContainer>
+    <Box
```

Comment:
Jeg tenkte jeg kunne legge igjen noen mer generelle kommentarer på hva jeg har gjort under her, og hvorfor det er som det er.

Målet med endringene i denne filen var å fikse to ting:
- Unngå en keyboard trap bug i `react-live`
- Forbedre  opplevelsen av å bruke dokumentasjonssiden med tastaturet.

Den viktigste utfordringen jeg måtte løse her var at om man tab-et seg inn i en LiveEditor, så var det ingen måte å komme seg ut av den på igjen med bare tastaturet. Dette er det man kjenner som en keyboard trap. Det er en bug i biblioteket vi bruker – react-live – som ikke har blitt fikset på over et år, og som det var på tide å finne en workaround for.

Heldigvis går det an å jobbe seg rundt. Man kan nemlig "slå av og på" editoren med en `disabled` prop, og det er jo en tilstand man kan tracke ganske enkelt. Man enabler den når man klikker på den, eller fokuserer på boksen rundt og trykker enter. Man disabler den ved å trykke utenfor, eller trykke escape.

Dere ser kanskje også har jeg har lagt til endel _after-styles her, som er en grønn boks som dukker opp når man fokuserer på boksen rundt editoren. Kanskje den trenger noe mer aria-magi? Men det ser ut til å funke ganske greit i alle fall.

Også er det et par litt irriterende workarounds her, som når man prøver å fokusere på et contenteditable-område. Da må man av en eller annen grunn slenge det fokus-statementet inn i en `setTimeout` for at det skal registrere seg på riktig måte.

Det er mye å skrive her, men test det gjerne selv for å se om flyten funker :)